### ISSUE TITLE
Daily motion video password protected cannot be downloaded

### ISSUE THREAD

**Issue by kscheung1228** [rg3/youtube-dl#11570]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.12.22*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [ x] I've **verified** and **I assure** that I'm running youtube-dl **2016.12.22**

### Before submitting an *issue* make sure you have:
- [ x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x ] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
```
$ youtube-dl -v --video-password ako1536 https://www.dailymotion.com/embed/video/k1E2urqdOSM9SEgeMVJ
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'--video-password', u'PRIVATE', u'https://www.dailymotion.com/embed/video/k1E2urqdOSM9SEgeMVJ']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.12.22
[debug] Python version 2.7.12 - Darwin-16.3.0-x86_64-i386-64bit
[debug] exe versions: none
[debug] Proxy map: {}
[dailymotion] k1E2urqdOSM9SEgeMVJ: Downloading webpage
WARNING: unable to extract comment count; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
ERROR: An extractor error has occurred. (caused by KeyError(u'title',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/Users/kashingcheung/anaconda/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 358, in extract
    return self._real_extract(url)
  File "/Users/kashingcheung/anaconda/lib/python2.7/site-packages/youtube_dl/extractor/dailymotion.py", line 155, in _real_extract
    self._check_error(metadata)
  File "/Users/kashingcheung/anaconda/lib/python2.7/site-packages/youtube_dl/extractor/dailymotion.py", line 288, in _check_error
    '%s said: %s' % (self.IE_NAME, info['error']['title']), expected=True)
KeyError: u'title'
Traceback (most recent call last):
  File "/Users/kashingcheung/anaconda/lib/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 694, in extract_info
    ie_result = ie.extract(url)
  File "/Users/kashingcheung/anaconda/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 364, in extract
    raise ExtractorError('An extractor error has occurred.', cause=e)
ExtractorError: An extractor error has occurred. (caused by KeyError(u'title',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.youtube.com/watch?v=BaW_jenozKc
- Single video: https://youtu.be/BaW_jenozKc
- Playlist: https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc

---

### Description of your *issue*, suggested solution and other information

Explanation of your *issue* in arbitrary form goes here. Please make sure the [description is worded well enough to be understood](https://github.com/rg3/youtube-dl#is-the-description-of-the-issue-itself-sufficient). Provide as much context and examples as possible.
If work on your *issue* requires account credentials please provide them or explain how one can obtain them.

**Comment by dstftw** [comment]
Duplicate of #9789.

**Comment by kscheung1228** [comment]
Not sure if I am missing anything

9789 just says it support the video password.
But from my log it seems its still not supported dailymotion ?

Last comment of 9789 says dailymotion may have changed format and there hasnt been any affirmative comment saying dailymotion video password is currently functional ?

**Comment by yan12125** [comment]
Password-protected videos are not resolved yet, so #9789 is still open
